google/ggrc-core
# GGRC-7990 Feature Request - Time overdue / time until due

# Issue description
Overdue Assessments are not highlighted.

# Steps to test the changes 
1. Open dashboard.
2. Open Assessments tab.
3. Look at the overdue assessments.

**Actual Result**:  Due date field is not highlighted.
**Expected Result**: Due date field is highlighted.

# Solution description 
Highlight the Due Date in the same way as End Date in Workflow.

# Sanity checklist 

# PR Review checklist
